lib/alpha_taxonomy/taxon_renamer.rb

```diff
@@ -0,0 +1,46 @@
+module AlphaTaxonomy
+  class TaxonRenamer
+    def initialize(logger: Logger.new(STDOUT), base_paths:)
+      @log = logger
+      @base_paths = base_paths
+    end
+
+    def run!
+      paths_and_content_ids = lookup_content_ids_by_base_paths
+
+      @base_paths.each do |base_path_pair|
+        content_id = paths_and_content_ids[base_path_pair[:from]]
+        title = normalised_title(base_path_pair[:to])
+
+        @log.info "-- Requesting base_path change for #{title}"
```

Comment:
Sorry, this is really picky but - I think I'd prefer `base_path_pair[:from]` here rather than `title`. I've just found the URL to be more readable (and easier to spot on the screen) when running these tasks in the console. Also, we've been 'handling' urls in the run up to this task executing - it's a bit less of a cognitive switch if we keep referring to these taxons by their URL.